app/dashboard/templates/profiles/activity_stats.html

```diff
@@ -0,0 +1,21 @@
+  <li>
+  last seen: <b>x days ago</b>
+  </li>
+  <li>
+  longest activity streak: <b>x days</b>
+  </li>
+  <li>
+  activity level: <b>high</b>
+  </li>
+  <li>
+  top skills: <b>python, django, javascript</b>
+  </li>
+  <li>
+  success pct: <b>100%</b>
```

Comment:
I just added a helper function for this `profile.success_rate` in https://github.com/gitcoinco/web/pull/5104

Reply:
thanks, this will be super helpful